import-export-cli/integration/api_test.go

```diff
@@ -30,6 +30,29 @@ import (
 
 const numberOfAPIs = 5 // Number of APIs to be added in a loop
 
+// Export an API from one environment and check the structure of the DTO whether it is similat to what is being
+// maintained by APICTL
+func TestExportApiCompareStruct(t *testing.T) {
+	apiPublisher := publisher.UserName
+	apiPublisherPassword := publisher.Password
+
+	apiCreator := creator.UserName
+	apiCreatorPassword := creator.Password
+
```

Comment:
shall we remove this extra line?

Reply:
I think it is better to keep this line to be more consistent because in almost every integration test we have followed the same convention. You can refer the tests in the same file (api_test.go) and get an idea.